.husky/.pre-commit.sh

```diff
@@ -7,8 +7,13 @@ STAGED_FILES=$(mktemp)
 
 # List staged .sol files ignoring deleted files
 git diff --cached --name-status -- '*.sol' | grep -v '^D' | cut -f2- > "$STAGED_SOL_FILES"
+
 # List all staged files ignoring deleted files
-git diff --cached --name-status | grep -v '^D' | cut -f2- > "$STAGED_FILES"
+
+# List only renamed files
+git diff --cached --name-status | grep -v '^D' | grep -E '^R' | cut -f3 > "$STAGED_FILES"
+# Append all staged files ignoring renamed and deleted files
+git diff --cached --name-status | grep -v '^D' | grep -E '^[^R]' | cut -f2 >> "$STAGED_FILES"
```

Comment:
Renamed files are already added on the line above and on line 16 we are appending only changed files. This is done because when a file is renamed the renamed file name is available at f3 in the diff. Before this change husky was failing because it was picking old file name at f2.

I just realized that I should do the same for `sol` files as well. will make this update.

Reply:
done here [8ccfb8b](https://github.com/solidity-labs-io/forge-proposal-simulator/pull/73/commits/8ccfb8bdca5ea5dfd58c6a8302f66c428b7cb757)